test/factories/ping_thing_recent_stats.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :ping_thing_recent_stat do
+    interval { 5 }
```

Comment:
I'd change that to trait named interval_5 similar to interval_60

Reply:
This field is obligatory, otherwise ping stat won't be created. I can add a trait additionally, but we need to have interval value here as well.